build/yaml/crd/nsx.vmware.com_ippools.yaml

```diff
@@ -173,9 +173,6 @@ spec:
                 type: array
               type:
                 description: Type defines the type of this IPPool, Public or Private.
-                enum:
```

Comment:
Suggest not to remove the validations on public and private enum. It would be useful.

Defining omitempty to Type field will allow user doesn't specify type field and will avoid the "spec.type: Unsupported value: "":" error.  In the meanwhile unmarshal the json to golang struct will got zero value of Type as well.

Reply:
ok